group_vars/all.yml

```diff
@@ -4,7 +4,7 @@ infrared_workspaces_dir: "{{ ansible_user_dir }}/.infrared/.workspaces"
 infrared_hosts_file: "{{ infrared_workspaces_dir }}/active/hosts"
 instackenv_file: "{{ playbook_dir }}/instackenv.json"
 undercloud_conf: "{{ ansible_user_dir }}/undercloud.conf"
-log_directory: "{{ playbook_dir }}/jetpack/logs"
+log_directory: ""{{ playbook_dir }}/logs"
```

Comment:
Don't need to change this

Reply:
I changed it from "{{ playbook_dir }}/ljetpack/logs"  to "{{ playbook_dir }}/logs" as it unnecessarily creates jetpack dir and then inside creates the log dir

Reply:
Ha, I see. Good catch.